Make gepetuto tests pass

TheoMF opened the pull request:
In the last commit  "make gepetuto test work" I modified the tp files manually so there may be mistakes. In the tp5 I commented a part of the notebook 5-2 as I don't know how to fix it, and the tp6 don't pass test, the error is attached
![tp6_error](https://github.com/Gepetto/supaero2023/assets/140606863/7ff8378a-bb9c-491f-ae25-e54557c54cda)

TheoMF committed:
import gepetuto.magic instead of magic_donotload

TheoMF committed:
change python version requirements for scipy

TheoMF committed:
update dependency, add poetry.lock file

TheoMF committed:
rename ipynbs to fit new naming convention

TheoMF committed:
update do_not_load_ commands name

TheoMF committed:
reformat generated files

TheoMF committed:
delete outputs

TheoMF committed:
make gepetuto test work

nim65s commented on tp0/generated/simple_path_planning_random_descent:
```diff
@@ -1,15 +1,19 @@
+    for i in range(100):
+        dq = qrand() * 0.1  # Choose a random step ...
+        q + dq  # ... apply
+        if dist(q) > dist(q + dq) and not coll(
+            q + dq
+        ):  # If distance decrease without collision ...
```
```suggestion
        if dist(q) > dist(q + dq) and not coll(q + dq):  
            # If distance decrease without collision ...
```

nim65s committed:
Update tp0/generated/simple_path_planning_random_descent

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

nim65s committed:
Merge branch 'main' into apply_gepetuto

nim65s committed:
CI: poetry run

fix: gepetuto: command not found

nim65s committed:
CI: pythonpath

fix:
Traceback (most recent call last):
  File "/home/runner/work/supaero2023/supaero2023/tp0/simple_path_planning.py", line 10, in <module>
    from utils.load_ur5_with_obstacles import Target, load_ur5_with_obstacles
ModuleNotFoundError: No module named 'utils'
